app/views/contributions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="section">
+  <h2>Request for Contribution</h2>
+  <h1 class="title"><%= contribution.type %>: <%= contribution.all_tags_unique.map(&:humanize).join(", ") %></h1>
+  <br>
+  <div class="box">
+    <p><strong><%= contribution.person.anonymize_name %></strong> - <%= contribution.person.anonymize_email %></p> 
+    <p class="subtitle"><%= contribution.description %></p>
+    <%= button_to "Claim this Ask", claim_ask_form_contribution_path(contribution), class: "button is-primary", method: :get %>
```

Comment:
As far as I can tell, an `Ask` is but one type of "contribution". I believe this bit is supposed to render differently based on the type of contribution. Additionally, the role of the person clicking interacting with the contribution changes depending on the type of contribtion. For example, if the contribution is an `Offer`, the user will be a `receiver` on the match, however if the contribution is an `Ask` the user will be a `provider`.

Reply:
Good catch! I can make that fix